modules/improvedigitalBidAdapter.js

```diff
@@ -124,6 +126,23 @@ export const spec = {
 
 registerBidder(spec);
 
+const convertBidFloorCurrency = (imp) => {
+  try {
+    const bidFloor = convertCurrency(
+      imp.bidfloor,
+      imp.bidfloorcur.toUpperCase(),
+      DEFAULT_CURRENCY,
+      false,
+    );
+    if (typeof bidFloor === 'number' && !isNaN(bidFloor)) {
```

Comment:
I think  it could happen, but that's just based off of one of the unit tests failing after removing the check due the bidFloor being null. I'll dig a little deeper into this.

Reply:
Found it - my `convertCurrency` mock wasn't accurate. I updated the mock and removed the check.